client/src/pages/ProfileDetails/ProfileDetails.tsx

```diff
@@ -0,0 +1,57 @@
+import useStyles from './useStyles';
+import { Box, Button, Typography, Select, MenuItem, OutlinedInput, Grid } from '@material-ui/core';
+import { useEffect } from 'react';
+import PinIcon from './PinIcon.png';
+import fetchProfile from '../../helpers/APICalls/fetchProfile';
+import BookingForm from './BookingForm/BookingForm';
+
+const ProfileDetails = (): JSX.Element => {
+  const classes = useStyles();
+
+  return (
+    <Box className={classes.profileDetailsPage}>
+      <Box className={classes.profileDetails}>
```

Comment:
You can apply all the styling to the Box component directly via its props ([docs](https://v4.mui.com/system/basics/#all-inclusive)). This is very beneficial since it reduces the context switch when working on the styling, but also to explicitly show some of the most impactful styles such as layout related styling options (e.g. `position/display`, `flexbox` attributes, `height/width`...). You can always decide to hide some of the minor styling into a class in order to avoid overcrowding the file but in my experience this doesn't happen very often.